docs/cookbook/html-conversion.md

```diff
@@ -0,0 +1,980 @@
+
+Second, we need to add the template styling and the main content area to the `main` block.
+
+<AposCodeBlock>
+
+::: v-pre
+
+```django
+{% extends "layout.html" %}
+
+{% import "fragments/header.html" as header %}
+
+{% block beforeMain %}
+  {{ super() }}
+  {% render header.headerArea(data.page) %}
+{% endblock %}
+
+  {% block main %}
+    <div class="container px-4 px-lg-5">
+      <div class="row gx-4 gx-lg-5 justify-content-center">
+        <div class="col-md-10 col-lg-8 col-xl-7">
+          {% area data.page, 'main' %}
+        </div>
+      </div>
+    </div>
+  {% endblock %}
+```
+
+:::
+
+<template v-slot:caption>
+modules/default-page/views/page.html
+</template>
+</AposCodeBlock>
+
+Next, we need to modify the schema fields of the default-page `index.js` file to add the data to the header and the main body of the page.
+
+<AposCodeBlock>
+
+```js
+module.exports = {
+  extend: '@apostrophecms/page-type',
+  options: {
+    label: 'Default Page'
+  },
+  fields: {
+    add: {
+      heading: {
+        type: 'string',
+        label: 'Heading'
+      },
+      subheading: {
+        type: 'string',
+        label: 'Subheading'
+      },
+      headerImage: {
+        type: 'area',
+        required: true,
+        options: {
+          widgets: {
+            '@apostrophecms/image': {}
+          },
+          max: 1
+        }
+      },
+      main: {
+        type: 'area',
+        options: {
+          widgets: {
+            '@apostrophecms/rich-text': {
+              toolbar: [
+                'styles',
+                '|',
+                'bold',
+                'italic',
+                'strike',
+                'link',
+                '|',
+                'bulletList',
+                'orderedList'
+              ],
+              styles: [
+                {
+                  tag: 'p',
+                  label: 'Paragraph (P)'
+                },
+                {
+                  tag: 'h3',
+                  label: 'Heading 3 (H3)'
+                },
+                {
+                  tag: 'h4',
+                  label: 'Heading 4 (H4)'
+                }
+              ]
+            },
+            '@apostrophecms/image': {},
+            '@apostrophecms/video': {}
+          }
+        }
+      }
+    },
+    group: {
+      basics: {
+        label: 'Basics',
+        fields: [
+          'title',
+          'heading',
+          'subheading',
+          'headerImage',
+          'main'
+        ]
+      }
+    }
+  }
+};
+
+```
+
+<template v-slot:caption>
+modules/default-page/index.js
+</template>
+</AposCodeBlock>
+
+To accommodate the 'Contact Us' page, we could also the widgets from the [form extension](https://apostrophecms.com/extensions/form-builder-3-x) in the `main` area. Since we are using the `default-page` which already exists in our project we don't need to modify either the `app.js` or `modules/@apostrophecms/page/index.js` files.
```

Comment:
```suggestion
To accommodate the 'Contact Us' page, we could also add the widgets from the [form extension](https://apostrophecms.com/extensions/form-builder-3-x) in the `main` area. Since we are using the `default-page` which already exists in our project we don't need to modify either the `app.js` or `modules/@apostrophecms/page/index.js` files.
```

> Since we are using the `default-page` which already exists in our project we don't need to modify either the `app.js` or `modules/@apostrophecms/page/index.js` files.

This sentence tripped me up a couple times. Got what I think you're saying after re-reading a couple times but wasn't completely clear first time around.

Reply:
Hopefully clarified.